test/acceptance/fake-server.ts

```diff
@@ -148,6 +148,41 @@ export function fakeServer(root, apikey) {
       result: {
         issues: [],
         issuesData: {},
+        depGraphData: {
+          schemaVersion: '1.2.0',
+          pkgManager: {
+            name: 'rpm',
+            repositories: [{ alias: 'rhel:8.2' }],
+          },
+          pkgs: [
+            {
+              id: 'docker-image|snyk/kubernetes-monitor@1.32.2',
+              info: {
+                name: 'docker-image|snyk/kubernetes-monitor',
+                version: '1.32.2',
+              },
+            },
+            {
+              id: 'acl@2.2.53-1.el8',
+              info: { name: 'acl', version: '2.2.53-1.el8' },
+            },
+          ],
+          graph: {
+            rootNodeId: 'root-node',
+            nodes: [
+              {
+                nodeId: 'root-node',
+                pkgId: 'docker-image|snyk/kubernetes-monitor@1.32.2',
+                deps: [{ nodeId: 'acl@2.2.53-1.el8' }],
+              },
+              {
+                nodeId: 'acl@2.2.53-1.el8',
+                pkgId: 'acl@2.2.53-1.el8',
+                deps: [],
+              },
+            ],
+          },
+        },
```

Comment:
I have a suggestion to reduce the size:
```suggestion
          pkgs: [
            {
              id: 'docker-image|foo@1.2.3',
              info: {
                name: 'docker-image|foo',
                version: '1.2.3',
              },
            },
          ],
          graph: {
            rootNodeId: 'root-node',
            nodes: [
              {
                nodeId: 'root-node',
                pkgId: 'docker-image|foo@1.2.3',
                deps: [],
              },
            ],
          },
        },
```